.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,42 @@
+name: ubuntu
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.3.x', '2.4.x', '2.5.x', '2.6.x' ]
```

Comment:
Is there a way to also migrate our jruby matrix entry?

Reply:
I will add jruby to ubuntu-rvm matrix.